invenio_drafts_resources/services/records/config.py

```diff
@@ -164,26 +167,33 @@ class RecordServiceConfig(RecordServiceConfigBase):
     links_item = {
         "self": ConditionalLink(
             cond=is_record,
-            if_=RecordLink("{+api}/records/{id}"),
-            else_=RecordLink("{+api}/records/{id}/draft"),
+            if_=RecordEndpointLink("records.read"),
+            else_=RecordEndpointLink("records.read_draft"),
         ),
         "self_html": ConditionalLink(
             cond=is_record,
-            if_=RecordLink("{+ui}/records/{id}"),
-            else_=RecordLink("{+ui}/uploads/{id}"),
+            if_=RecordEndpointLink("invenio_app_rdm_records.record_detail"),
```

Comment:
as i understand the `RecordEndpointLink` `invenio_app_rdm_records` is [this](https://github.com/inveniosoftware/invenio-app-rdm/blob/master/invenio_app_rdm/records_ui/views/__init__.py#L72)  blueprint. which means that we add an implicit dependency to `invenio-app-rdm` here.

in my opinion this is not good. i thought `invenio-drafts-resources` is independent of `app-rdm`.